### ISSUE TITLE
Fatal error on compile (for install)

### ISSUE THREAD

**Issue by benduckwdesign** [Jonimoose/TITools#1]
I wanted to start doing some ASM coding for my TI-84+ CSE, but it seems like something isn't quite right.

Help is appreciated, since I've already installed previous libraries (libticables, libticalcs2, libticonv, libtifiles, glib...) 

```
cd src && /Applications/Xcode.app/Contents/Developer/usr/bin/make install
gcc -I.. -I. -g -O2 -W -Wall -Wwrite-strings  -I/usr/local/include/tilp2 -I/usr/local/Cellar/glib/2.46.2/include/glib-2.0 -I/usr/local/Cellar/glib/2.46.2/lib/glib-2.0/include -I/usr/local/opt/gettext/include -DHAVE_CONFIG_H -c ./tidump.c
./tidump.c:28:10: fatal error: 'cmd73.h' file not found
#include <cmd73.h>
         ^
1 error generated.
make[1]: *** [tidump.o] Error 1
make: *** [install] Error 2
```

**Comment by Jonimoose** [comment]
That file can be gotten from the libticalcs source. I had thought upstream was including that in the include paths but I'll double check with them and try to commit a fix. In the mean time copying that file where gcc will see it should do the trick.

**Comment by benduckwdesign** [comment]
EDIT: That looks terrible, so I'm uploading a picture.

![screen shot 2016-05-09 at 3 51 46 pm](https://cloud.githubusercontent.com/assets/5730541/15126026/0a48d11c-15fe-11e6-9fba-dce61daa1361.png)

Seems like something else goes totally wrong after adding the missing header file.

**Comment by Jonimoose** [comment]
Per upstream the header has been replaced by cmdz80.h so if you edit the code to be #include <cmdz80.h> and I'll see if I can get time tonight to fix it in the repo and do a new release.

**Comment by Jonimoose** [comment]
This should be fixed by d353be7cab61b1358eba74ef7f6a472d15253513
Marking this closed unless you see futher issues.
